src/k8s-engine/client.js

```diff
@@ -206,6 +206,14 @@ class KubeClient {
         address ||= this.#server.address();
         return address.port;
     }
+
+    async cancelForwardPort() {
```

Comment:
I just realized that my editor was set up wrong and wasn't obeying the `.editorconfig`; Will need to make a follow-up PR to fix all that, sorry.